src/test/ui/proc-macro/invalid-punct-ident-2.rs

```diff
@@ -1,4 +1,5 @@
 // aux-build:invalid-punct-ident.rs
+// needs-unwind
```

Comment:
Ditto here for panicking proc macro comment. The rest of these all appear to be proc macro tests too, they should all have the comment too.